release_notes/v6_1_1.html.md

```diff
@@ -0,0 +1,5 @@
+# v6.1.1 (2019-02-21)
+### Bug Fixes
+* Fixes the error in the type definition files
```

Comment:
Should this not be a little more descriptive of the specific bug / error it's fixing?

Reply:
Added some more to it